tests/golos.publication_tests.cpp

```diff
@@ -206,7 +206,7 @@ BOOST_FIXTURE_TEST_CASE(create_message, golos_publication_tester) try {
     BOOST_TEST_MESSAGE("--- checking that another user can create a message with the same permlink.");
     BOOST_CHECK_EQUAL(success(), post.create_msg(N(chucknorris), "permlink"));
 
-    auto id = hash64("permlink");
+    auto id = 1;
     check_equal_post(post.get_message(N(brucelee), id), _test_msg);
```

Comment:
It's seems like `post.get_message(...)` and `post.get_content(...)` should get permlink instead of id